### ISSUE TITLE
Sharing to other users

### ISSUE THREAD

**Issue by matmerce** [dmreagan/collectome#68]
Is it possible to make a private collection, but be able to make it accessible to people you send the link to people? That would be cool.

**Comment by dmreagan** [comment]
No, private is private to just the owner of the collection. But in the future we could consider having functionality to add collaborators to a private collection.
